open-ce/utils.py

```diff
@@ -106,6 +106,17 @@ class Argument(Enum):
                                              "should be relative to the directory containing open-ce. Only files "
                                              "within the open-ce directory and local_files will be visible at "
                                              "build time."))
+    CONDA_ENV_FILE = (lambda parser: parser.add_argument(
+                                        '--conda_env_file',
+                                        type=str,
+                                        help='Location of conda environment file.' ))
+
+    LOCAL_CONDA_CHANNEL = (lambda parser: parser.add_argument(
```

Comment:
We could probably reuse the normal channels argument instead of creating the local_conda_channel argument.

Reply:
Yes, I could but I thought normal channels is `--channels` which could be local conda directory or any other channel like PowerAI's or Anaconda's channel. Here, I wanted to be explicit that this is local conda channel which will be copied to the docker image.

Reply:
Yeah, makes sense.